### PR TITLE
Lucuma Core 73

### PR DESCRIPTION
**Pull request by swalker2m**
Updates to `lucuma-core.0.73.0`, which includes `SignalToNoise`.  Fixes the schema fetch script.

### CODE REVIEW

File: fetchODBSchema.sh [hunk 1]
```diff
@@ -1,5 +1,51 @@
...
+# fetchODBSchema.sh
+#
+# Usage: sh fetchODBSchema.sh [local|staging]
+#
+# Fetches the current version of the ODB schema from either a 'local' ODB
+# instance running on localhost, or else from the 'staging' ODB.  Places it
+# in the proper place in `lucuma-schemas` for use in generating code.
+#
+
+KEY_MESSAGE=$(cat << END
```
Comment (cquiroz) [inline fetchODBSchema.sh:13]: This is way more profession than what we had 😄  thanks

Comment (rpiaggio) [reply fetchODBSchema.sh:13]: Nice!

Comment (toddburnside) [reply fetchODBSchema.sh:13]: Yeah. Very nice. Thanks for the improvements!

**Review by rpiaggio — APPROVED** [review]
LGTM

**Pull request merged**